src/cms/templates/pois/poi_list.html

```diff
@@ -37,13 +50,13 @@ <h2 class="heading font-normal mb-2">{% trans 'Locations on map' %}</h2>
 </div>
 
 <div class="table-listing">
-    <table class="w-full mt-4 rounded border border-solid border-gray-400 shadow bg-white table-fixed">
+    <table id="pois-list" class=" sort w-full mt-4 rounded border border-solid border-gray-400 shadow bg-white table-fixed">
         <thead>
             <tr class="border-b border-solid border-gray-400">
                 <th class="text-sm text-left uppercase py-3 pl-4 pr-2 min">{% trans 'ID' %}</th>
                 <th class="text-sm text-left uppercase py-3 pr-2 min">{% trans 'Version' %}</th>
-                <th class="text-sm text-left uppercase py-3">{% trans 'Status' %}</th>
-                <th class="text-sm text-left uppercase py-3">{% trans 'Title in' %} {{ language.translated_name }}</th>
+                <th class="text-sm text-left uppercase py-3 px-2">{% trans 'Status' %}</th>
```

Comment:
```suggestion
                <th class="text-sm text-left uppercase py-3 pr-2">{% trans 'Status' %}</th>
```
This is going into the wrong direction. Please do not add the class `px-2` to all classes, but rather `pr-2`. The reason is that we want to align all table fields to their left edge. If both the table head and the table cell have the same padding, they are indeed aligned, but together with the padding of the previous cells, we have a total padding of 4 between the cells and this is too much. Just add a padding-right of 2 to all cells (`pr-2`) and remove the `px-2` where you find it.